lib/splicer/dns_made_easy/record.rb

```diff
@@ -2,7 +2,10 @@ module Splicer
   module DnsMadeEasy
 
     class Record
-      attr_reader :id, :type, :value, :ttl, :data
+      attr_reader :name, :value, :id, :type, :dynamicDns,
```

Comment:
Change `attr_reader` to `attr_accessor` and you can get rid of the `add_attribute` method.